docs/_snippets/manual-setup-express.mdx

```diff
@@ -1 +1,187 @@
+---
+
+<Accordion defaultOpen title="Don't have an Express project yet to add Trigger.dev to? No problem, you can complete the Manual Setup using a blank Express project:">
+Create a blank project by creating a new Express application:
+
+```bash
+npm init -y
+npm install express
+```
+
+Trigger.dev works with Express applications.
+
+</Accordion>
+
+## Installing Required Packages
+
+Start by installing the necessary packages in your Express.js project directory. You can use npm, pnpm, or yarn as your package manager.
+
+<CodeGroup>
+
+```bash npm
+
+npm install @trigger.dev/sdk express @trigger-dev/express
+
+
+```
+
+```bash pnpm
+pnpm install @trigger.dev/sdk express @trigger-dev/express
+
+
+```
+
+```bash yarn
+yarn add @trigger.dev/sdk express @trigger-dev/express
+
+```
+
+</CodeGroup>
+
+<br />
+
+<Note>Ensure that you execute this command within a Express project.</Note>
+
+## Obtaining the Development API Key
+
+To locate your development API key, login to the [Trigger.dev
+dashboard](https://cloud.trigger.dev) and select the Project you want to
+connect to. Then click on the Environments & API Keys tab in the left menu.
+You can copy your development API Key from the field at the top of this page.
+(Your development key will start with `tr_dev_`).
+
+## Adding Environment Variables
+
+Create a `.env.local` file at the root of your project and include your Trigger API key and URL like this:
+
+```bash
+
+TRIGGER_API_KEY=ENTER_YOUR_DEVELOPMENT_API_KEY_HERE
+TRIGGER_API_URL=https://cloud.trigger.dev
+
+```
+
+Replace `ENTER_YOUR_DEVELOPMENT_API_KEY_HERE` with the actual API key obtained from the previous step.
+
+## Configuring the Trigger Client
+
+To set up the Trigger Client for your project, follow these steps:
+
+1. **Create Configuration File:**
+
+   In your project directory, create a configuration file named `trigger.ts` or `trigger.js`, depending on whether your project uses TypeScript (`.ts`) or JavaScript (`.js`).
+
+2. **Choose Directory:**
+
+   Depending on your project structure, choose the appropriate directory for the configuration file. If your project uses a `src` directory, create the file within it. Otherwise, create it directly in the project root.
+
+3. **Add Configuration Code:**
+
+   Open the configuration file you created and add the following code:
+
+   ```typescript
+   // trigger.ts (for TypeScript) or trigger.js (for JavaScript)
+
+  // trigger.ts
+
+import express, { Express } from 'express';
+import { TriggerClient } from '@trigger.dev/sdk';
+
+const app: Express = express();
+const port: number = process.env.PORT ? parseInt(process.env.PORT) : 3000;
+
+const triggerClient: TriggerClient = new TriggerClient({
+  id: 'my-express-app',
+  apiKey: process.env.TRIGGER_API_KEY || '',
+  apiUrl: process.env.TRIGGER_API_URL || '',
+});
+
+// Your Express.js app configuration and routes go here
+
+app.listen(port, () => {
+  console.log(`Server is running on port ${port}`);
+});
```

Comment:
You need to actually use our @trigger.dev/express package somewhere to create a route